File: HackerRank-30-Days-Of-Code/JavaScript/3.js
const N = parseInt(readLine(), 10);
if (N % 2 == 0){
    if (N >= 2 && N < 6){
        console.log("Not Weird")
    }
    if (N >= 6 && N < 21){
        console.log("Weird");
    }
    if(N>20) {
        console.log("Not Weird");
    }
}else {
    console.log("Weird");
}

File: HackerRank-30-Days-Of-Code/Python/3.py
#!/bin/python3

import math
import os
import random
import re
import sys



if __name__ == '__main__':
    N = int(input())

    if (N % 2 == 1) or ( N >= 5 and N <= 21 ):
        print("Weird")
    else:
        print("Not Weird")
